integration-tests/tests/schema/metadata.spec.ts

```diff
@@ -0,0 +1,152 @@
+import { getServiceHost } from 'testkit/utils';
+import type { SchemaBuilderApi } from '@hive/schema';
+import { createTRPCProxyClient, httpLink } from '@trpc/client';
+
+const host =
```

Comment:
nit: should this go somewhere into the test kit? 😄

Reply:
possibly... I dont see anywhere in the codebase where we use `SCHEMA_SERVICE_HOST_OVERRIDE`... If anything I assume `RUN_AGAINST_LOCAL_SERVICES` is more practical for us to use for overriding the hosts.
I'd be in favor of removing the env var condition in this file and in the `contracts.spec.ts` before moving this into the test kit.